src/components/RequestReview/RequestReview.tsx

```diff
@@ -109,12 +104,12 @@ export const RequestReview: React.FunctionComponent<IRequestReviewProps> = (prop
                     }
                     {request && !request.isReadOnly(user, roles) &&
                         <Button className="float-right mr-2" variant="warning"
-                            onClick={(e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => redirect(e, `/Requests/Edit/${request.Id}`)}
+                            onClick={(e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => redirect(`/Requests/Edit/${request.Id}`, e)}
                         >
                             Edit Request
                         </Button>}
                     <Button className="float-right mr-2" variant="secondary"
-                        onClick={(e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => redirect(e, "/Requests")}
+                        onClick={(e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => redirect("/Requests", e)}
                     >
```

Comment:
Similar to other area - should these be Buttons wrapped in the Link element?
<Link to...><Button ...></Button></Link>